### PR TITLE
Update Quick-Start-Guide.rst

### PR DESCRIPTION
**Pull request by kimika88**
add tested by list
Signed-off-by: Yixuan Li yixuan.li@seagate.com

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: doc/Quick-Start-Guide.rst [hunk 1]
```diff
@@ -167,6 +167,8 @@ The files will be generated at doc/html/ folder.
 
 
 Tested by:
+- September 20, 2021: Yixuan Li (yixuan.li@seagate.com) in Red Hat Enterprise Linux Server release 7.7 (Maipo)
+
```
Comment (mehjoshi) [inline doc/Quick-Start-Guide.rst:171]: @kimika88, Please add the git hash here similar to what we see in the line below.

Comment (kimika88) [reply doc/Quick-Start-Guide.rst:171]: 5aac28633a149d2c7e6f8d4c502d80dabf7ebb7e

File: doc/Quick-Start-Guide.rst [hunk 2]
```diff
@@ -167,6 +167,8 @@ The files will be generated at doc/html/ folder.
 
 
 Tested by:
+- September 20, 2021: Yixuan Li (yixuan.li@seagate.com) in Red Hat Enterprise Linux Server release 7.7 (Maipo)
```
Comment (mehjoshi) [inline doc/Quick-Start-Guide.rst]: - September 20, 2021: Yixuan Li (yixuan.li@seagate.com) in Red Hat Enterprise Linux Server release 7.7 (Maipo)
(#5aac28633a149d2c7e6f8d4c502d80dabf7ebb7e)

**Pull request merged**